tests/testthat/test_get_biomart_pfam_merge.R

```diff
@@ -0,0 +1,12 @@
+context("Test biomart pfam data")
+
+ensembl <- biomaRt::useMart(biomart = "ENSEMBL_MART_ENSEMBL",dataset = "hsapiens_gene_ensembl",host="ensembl.org")
+
+download.file("http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/pfamDesc.txt.gz","pfamDesc.txt.gz")
+download.file("http://hgdownload.soe.ucsc.edu/goldenPath/hg38/database/ucscGenePfam.txt.gz","ucscGenePfam.txt.gz")
```

Comment:
if we download these - probably set the destination to a temporary data folder? Again - the cache solution would be more elegant